src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -849,6 +1039,14 @@ end
     return prim2cons(SVector(rho, v1, v2, v3, p, B1, B2, B3, psi), equations)
 end
 
+@inline function isValidState(cons, equations::IdealGlmMhdEquations2D)
```

Comment:
```suggestion
@inline function is_valid_state(cons, equations::IdealGlmMhdEquations2D)
```
I changed this in the last PR. Sorry.